wagtail/contrib/forms/templates/wagtailforms/bulk_actions/list_form_submissions.html

```diff
@@ -0,0 +1,57 @@
+{% load i18n %}
+<table class="listing">
+    <thead>     
+        <tr>
+            <th id="submit_time" class="">
+                Submission date
+            </th>
+            <th id="subject" class="">
+                Subject
+            </th>
+
+            <th id="your_name" class="">
+                Your Name
+            </th>
+
+            <th id="your_email" class="">
+                Your Email
+            </th>
+
+            <th id="purpose" class="">
+                Purpose
+            </th>
+
+            <th id="body" class="">
+                Body
+            </th>
+        </tr>
+    </thead>
+{% for item in items %}
+    <tbody>
+        <tr>
+            <td>
+                {{item.item.submit_time}}
```

Comment:
Pretty sure these columns should not be hard-coded. Except maybe submission date.

Reply:
Yes I understand as all forms will not be having same fields

Reply:
@lb-  I am working for this one.

Reply:
I have made a PR resolving this, please verify this one.